core/src/main/java/org/frankframework/jdbc/XmlQuerySender.java

```diff
@@ -175,22 +174,16 @@ private void fillParameter() throws SenderException {
 				}
 				parameter = new Timestamp(nDate.getTime());
 			} else if (type.equalsIgnoreCase(TYPE_BLOB)) {
-				if (!getDbmsSupport().mustInsertEmptyBlobBeforeData()) {
-					parameter = value.getBytes();
-				}
+				parameter = value.getBytes();
 			} else {
-				if (!(type.equalsIgnoreCase(TYPE_CLOB) && getDbmsSupport().mustInsertEmptyClobBeforeData()) && !type.equalsIgnoreCase(TYPE_FUNCTION)) {
+				if (!type.equalsIgnoreCase(TYPE_FUNCTION)) {
```

Comment:
Inserting empty BLOB was something for Oracle, but no longer an issue now. So code is simplified.